firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java

```diff
@@ -766,6 +768,19 @@ public void getInfo_hitsThrottleLimit_throttledStatus() throws Exception {
         .isEqualTo(firstFetchedContainer.getFetchTime());
   }
 
+  @Test
+  public void customSignals_updated_onSubsequentFetch() throws Exception {
+    Map<String, String> customSignals =
+        ImmutableMap.of(
+            "subscription", "premium",
+            "age", "20");
+    sharedPrefsClient.setCustomSignals(customSignals);
+    fetchCallToHttpClientUpdatesClockAndReturnsConfig(firstFetchedContainer);
+    fetchHandler.fetch();
+
+    verifyCustomSignals(customSignals);
```

Comment:
+1 to Ashish's comment, I would also expect to see the actual assertion in this test